### PR TITLE
Navigation

### PR DESCRIPTION
**Pull request by kraviecdev**
new branch for compare

### CODE REVIEW

File: src/common/Navigation/styled.js [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+        flex-grow: 0;
+    };
+`;
+
+export const NavigationTitle = styled.h1`
+    margin: 0;
+    font-weight: 500;
+    line-height: 40px;
+    text-transform: capitalize;
+    letter-spacing: -1.5px;
```
Comment (maxnatalia) [inline src/common/Navigation/styled.js]: Here, in Figma we have -0.5px letter-spacing

File: src/common/Navigation/styled.js [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+export const NavigationWrapper = styled.div`
+    margin: auto;
+    display: flex;
+    align-items: center;
+    flex-wrap: wrap;
+    max-width: 1330px;
+    padding: 0 16px;
+
+    @media(max-width: ${({ theme }) => theme.breakpoint.smallScreen}px) {
+        max-width: ${({ theme }) => theme.breakpoint.smallScreen}px;
```
Comment (maxnatalia) [inline src/common/Navigation/styled.js]: unnecessary code duplicated `max-width: ${({ theme }) => theme.breakpoint.smallScreen}px;`

**Review by maxnatalia — APPROVED** [review]
Great! It seems OK, but I've had some cosmetics issues, but generally I approve these changes.

File: src/common/Navigation/styled.js (hunk 2 above)
Comment (kraviecdev) [reply src/common/Navigation/styled.js]: deleted unnecessary code, make nav buttons smaller for the smallest devices

**Pull request merged**